Readme.md

```diff
@@ -392,6 +393,14 @@ canvas.toDataURL('image/jpeg', {...opts}, (err, jpeg) => { }) // see Canvas#crea
 canvas.toDataURL('image/jpeg', quality, (err, jpeg) => { }) // spec-following; quality from 0 to 1
 ```
 
+### CanvasRenderingContext2D#textTracking
```

Comment:
Is this present in the spec for `CanvasRenderingContext2D`, or supported by any browsers?

Reply:
No, this would be an extension since HTML Canvas makes no provisions for setting letterspacing. See @zbjornson's summary of current browser behaviors here: https://github.com/Automattic/node-canvas/issues/1014#issuecomment-488970083